mcux/drivers/imx/fsl_adc.h

```diff
@@ -20,7 +20,7 @@
  * Definitions
  ******************************************************************************/
 /*! @brief ADC driver version */
-#define FSL_ADC_DRIVER_VERSION (MAKE_VERSION(2, 0, 3)) /*!< Version 2.0.3. */
+#define FSL_ADC_DRIVER_VERSION (MAKE_VERSION(2, 0, 2)) /*!< Version 2.0.2. */
```

Comment:
Please check that we're not reverting to older drivers. I suspect that we probably only want to import the devices and boards folders, and skip everything in drivers.

Reply:
revised